demo/Alsa cpp/compile.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
```

Comment:
Thanks for making this cross-platform. Maybe we can use Cmake? You can actually write the whole thing in few lines of code then :)

Reply:
downloading it now.. see how long it takes me to work this out :)

Reply:
If too much headache, don't bother. But when you learn it you will use it forever!